build/dependencies.props

```diff
@@ -1,11 +1,12 @@
-<Project>
+﻿<Project>
   <PropertyGroup>
     <AspNetCoreIntegrationTestingVersion>0.4.0-*</AspNetCoreIntegrationTestingVersion>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <CoreFxVersion>4.4.0-*</CoreFxVersion>
     <DotnetDebToolVersion>2.0.0-preview2-*</DotnetDebToolVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
-    <MoqVersion>4.7.1</MoqVersion>
+    <MoqVersion>4.7.49</MoqVersion>
+    <NETStandardImplicitPackageVersion>2.0.0-*</NETStandardImplicitPackageVersion>
```

Comment:
We used to have this so I think it should be safe: https://github.com/aspnet/MetaPackages/blob/b3b986bfa9a567e24df7267f10690a95b9493942/build/dependencies.props